MergeSharpWebAPI/Program.cs

```diff
@@ -170,46 +131,66 @@ private static async Task Main(string[] args)
             await connection.StartAsync();
             Console.WriteLine("Connection started");
 
-            //myLWWSetService.AddElement(1, 11);
-            //myLWWSetService.AddElement(1, 22);
-            //myLWWSetService.AddElement(1, 33);
-            //myLWWSetService.RemoveElement(1, 11);
-            //myLWWSetService.RemoveElement(1, 22);
-
-            //var byteMsg = myLWWSetService.Get(1).LwwSet.GetLastSynchronizedUpdate().Encode();
-            //MergeSharp.LWWSetMsg<int> lwwMsg = new();
-            //lwwMsg.Decode(byteMsg);
-            //Console.WriteLine("addset: " + string.Join(",", lwwMsg.addSet.Keys.ToList()));
-            //Console.WriteLine("removeset: " + string.Join(",", lwwMsg.removeSet.Keys.ToList()));
-
+            //Console.WriteLine("Raised RecieveMessage event on all clients");
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error occured when connecting to server:");
+            Console.WriteLine(ex.Message);
+        }
+    }
 
+    private static List<Node> translateKeystoNodes()
```

Comment:
I think it'd be better to keep all of the translation logic within the controller. On L115 you can send the controller the graph as is, then in the Controller let it handle translating the Graph into the JSON format with keys for the frontend. That way we keep the convention that the Controller acts as the translator between Guids and keys.


No need to address in this pr tho.